act-rules-format.bs

```diff
@@ -74,96 +74,96 @@ The actual definition of specific accessibility requirements is beyond the scope
 Limitations, Assumptions or Exceptions {#structure-limitations-assumptions-exceptions}
 -------------------------------------------------------------------
 
-An ACT Rule MUST list any limitations, assumptions or any exceptions for the test, the test environment, technologies being used or the subject being tested. For example, a rule for Success Criterion 1.4.1: Use of Color has to make an assumption that CSS is used to make a link visually evident - typically by using CSS background, border, color, font, or text-decoration properties.
+An ACT Rule MUST list any limitations, assumptions or any exceptions for the test, the test environment, technologies being used or the subject being tested. For example, a rule for [WCAG 2.0 Success Criterion 1.4.1: Use of Color](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-without-color) [[WCAG20]] has to make an assumption that CSS is used to make a link visually evident - typically by using CSS background, border, color, font, or text-decoration properties.
 
 
 Accessibility Support {#acc-support}
 ===========================
 
-ACT Rules are designed to test accessible uses of a web technology. However, not every part of a web technology is implemented in all assistive technologies a website may need to support. The concept of [accessibility supported](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef) use of a Web technology is defined in [[WCAG20]]. Because of this, ACT Rules are not necessarily applicable in all test scenarios. For instance, a web page that has to work in assistive technologies that have no WAI-ARIA support, wouldn’t be tested with an ACT Rule that relies on WAI-ARIA support, since this could lead to false positive results.
```

Comment:
This paragraph still contains a reference to `WAI-ARIA-1.1`; should be `WAI-ARIA`.

Reply:
Good catch. Thanks.

Reply:
Waiting for confirmation from group if we should link to TR or overview page.